src/runtime_src/Makefile

```diff
@@ -0,0 +1,88 @@
+# Build xilinxopencl.so for ARM
```

Comment:
Let's rename to arm.mk or something else.  Point is this is not a generic Makefile from build runtime_src which you might otherwise be led to believe.